docs/user-guide/custom-themes.md

```diff
@@ -302,7 +316,13 @@ full search implementation to your theme.
 <h1 id="search">Search Results</h1>
 
 <form action="search.html">
-  <input name="q" id="mkdocs-search-query" type="text" class="search_input search-query ui-autocomplete-input" placeholder="Search the Docs" autocomplete="off">
+  <input
+      name="q"
+      id="mkdocs-search-query"
+      type="text"
+      class="search_input search-query ui-autocomplete-input"
+      placeholder="Search the Docs"
+      autocomplete="off">
```

Comment:
We could remove some of it. The CSS class is irrelevant. I copied it from a theme which is why it is longer than we need.